ecl2df/faults.py

```diff
@@ -92,7 +113,7 @@ def faults_main(args) -> None:
     faults_df = df(deck)
     write_dframe_stdout_file(
         faults_df,
-        args.output,
+        args,
```

Comment:
The function should not be given all of the args, you only want to provide the absolute minimum number of inputs it needs.